### PR TITLE
tests, integration-tests: port leftover install tests to spread

### PR DESCRIPTION
**Pull request by fgimenez**
Requires #1381

### CODE REVIEW

File: tests/install-errors/task.yaml [hunk 1]
```diff
@@ -1,4 +1,22 @@
 summary: Checks for cli errors installing snaps
+environment:
+  SIDELOAD_SNAP_NAME: basic-binaries
+  STORE_SNAP_NAME: hello-world
+  SNAP_FILE: "./$[SIDELOAD_SNAP_NAME]_1.0_all.snap"
+
+prepare: |
```
Comment (zyga) [inline tests/install-errors/task.yaml]: I love how those tests look like. If you recall our conversation with @elopio at the sprint I argued we should do something exactly like this. This is brilliant.

**Comment by zyga** [comment]
Looks good, +1

File: tests/install-errors/task.yaml (hunk 1 above)
Comment (fgimenez) [inline tests/install-errors/task.yaml]: @zyga yep :) exactly that conversation is the inspiration for this, spread made it possible. i'm very glad you like it, thanks a lot!

**Comment by mvo5** [comment]
Looks good, however there is a spread failure in here.

**Comment by fgimenez** [comment]
@mvo5 thanks! yes, a merge with master was missing, it should be fixed now

**Pull request merged**